lib/MqttService/src/MqttService.cpp

```diff
@@ -163,9 +163,10 @@ void MqttService::process()
                 /* Authentication necessary? */
                 if (false == m_user.isEmpty())
                 {
-                    LOG_INFO("Connect to %s as %s with %s.", m_url.c_str(), m_user.c_str(), m_hostname.c_str());
+                    LOG_INFO("Connect to %s as %s : %s with %s.", m_url.c_str(), m_user.c_str(), m_password.c_str(), m_hostname.c_str());
 
-                    isConnected = m_mqttClient.connect(m_hostname.c_str(), m_user.c_str(), m_password.c_str());
+                    //isConnected = m_mqttClient.connect(m_hostname.c_str(), m_user.c_str(), m_password.c_str());
+                    isConnected = m_mqttClient.connect(m_hostname.c_str(), "fdrs", "data"); //just for testing

```

Comment:
Any concerns about skipping this whole file? :-)
Note, MQTT is still under development. Locally I have further changes, but not ready to push yet.

Reply:
you are right - this was just a test - I wonder why it landed in the PR